verilog/CST/expression.h

```diff
@@ -81,6 +81,12 @@ const verible::Symbol* GetConditionExpressionTrueCase(const verible::Symbol&);
 // Returns the false-case expression of a kConditionExpression.
 const verible::Symbol* GetConditionExpressionFalseCase(const verible::Symbol&);
 
+// Returns the operator of a kUnaryPrefixExpression
```

Comment:
Please add tests for the new functions in expression_test.cc.

Reply:
Done. Let me know if they're thorough enough.